com.github.actionless.oomox.json

```diff
@@ -0,0 +1,111 @@
+{
+    "app-id": "com.github.actionless.oomox",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "oomox-gui",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=~/.themes:create",
+        "--filesystem=~/.icons:create",
+        "--socket=x11"
+    ],
+    "modules": [
+        {
+            "name": "sassc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cd ./deps/libsass ; autoreconf -i ; ./configure --prefix=/app ; make ; make DESTDIR=/ install",
+                "cd ./deps/sassc ; autoreconf -i ; ./configure --prefix=/app ; make ; make DESTDIR=/ install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/sass/libsass/archive/3.4.7.tar.gz",
+                    "sha256": "855c40528b897d06ae4d24606c2db3cd09bb38de5b46b28e835f9d4fd4d7ab95",
+                    "dest": "./deps/libsass/"
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/sass/sassc/archive/3.4.2.tar.gz",
+                    "sha256": "ad805f2d404d17cf2980c8079a7413cd58d2f2085120167997b85420a722e079",
+                    "dest": "./deps/sassc/"
+                }
+            ]
+        },
+        {
+            "name": "imagemagick",
+            "config-opts": [
+                "--enable-static=no",
+                "--disable-docs",
+                "--disable-deprecated",
+                "--without-autotrace",
+                "--without-bzlib",
+                "--without-djvu",
+                "--without-dps",
+                "--without-fftw",
+                "--without-fontconfig",
+                "--without-fpx",
+                "--without-freetype",
+                "--without-gvc",
+                "--without-jbig",
+                "--without-jpeg",
+                "--without-lcms",
+                "--without-lzma",
+                "--without-magick-plus-plus",
+                "--without-openexr",
+                "--without-openjp2",
+                "--without-pango",
+                "--without-raqm",
+                "--without-tiff",
+                "--without-webp",
+                "--without-wmf",
+                "--without-x",
+                "--without-xml",
+                "--without-zlib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.imagemagick.org/download/ImageMagick-7.0.7-22.tar.xz",
+                    "sha256": "49de9e08ea255a1f939158d85d50dfa29285bccbdcb7fee0fe4309061d438489"
+                }
+            ]
+        },
+        {
+            "name": "oomox",
+            "buildsystem": "simple",
+            "build-commands": [
+                "rm -r ./plugins/{oomoxify,theme_materia}",
```

Comment:
Since you are upstream surely you could clean up this build process?

Reply:
that's needed only for flatpack because i can't get imagemagick or root access there

it's not a case even with docker distribution, not speaking about classical packages for distribution

Reply:
Typically build systems have configuration options to disable features or set prefixes. Using `rm` and `sed` on software you maintain is just ugly.

Reply:
but the thing is what sed-ing is needed only for flatpack because of its quirk with /usr/

regarding plugins it could make sense however since they are just directories it doesn't seems reasonable to me to write and maintain some additional code which in fact will be parsing and checking some arguments and in the end just removing plugins' directories

Reply:
> but the thing is what sed-ing is needed only for flatpack because of its quirk with /usr/

Setting a custom prefix has been a core part of build systems for many decades.